lib/Form/Processor/ContentFormProcessor.php

```diff
@@ -153,7 +156,12 @@ public function processCreateDraft(FormActionEvent $event)
      */
     private function saveDraft(ContentStruct $data, $languageCode)
     {
+        $mainLanuageCode = $this->resolveMainLanguageCode($data);
```

Comment:
Typo on `$mainLanguageCode`.

Reply:
Argh, happens to me on word *language* constantly ;)